asn1crypto/algos.py

```diff
@@ -454,6 +454,15 @@ class Pbes1Params(Sequence):
     ]
 
 
+class CcmParams(Sequence):
+    # https://tools.ietf.org/html/rfc5084
+    # aes_ICVlen: 4 | 6 | 8 | 10 | 12 | 14 | 16
+    _fields = [
+        ('aes_nonce', OctetString),
+        ('aes_ICVlen', Integer),
```

Comment:
So far all of the field names in the library have been all lowercase (if I recall correctly). For consistency, can we make this lowercase also?